### PR TITLE
Major changes in design using Bootstrap 3.3.1 and FontAwesome.

### PR DESCRIPTION
**Pull request by thegeorgeous**
Font set to Roboto and color scheme changed to reflect Django Project
I think we need to add more details about the project.
django non-rel 2014-12-30 23-32-12

### CODE REVIEW

**Comment by thegeorgeous** [comment]
@aburgel, what do you think?

File: index.html [hunk 1]
```diff
@@ -1,53 +1,101 @@
...
+        <script src="js/vendor/bootstrap.min.js"></script>
+
+        <!-- Google Analytics: change UA-XXXXX-X to be your site's ID. -->
+        <script>
+            (function(b,o,i,l,e,r){b.GoogleAnalyticsObject=l;b[l]||(b[l]=
+            function(){(b[l].q=b[l].q||[]).push(arguments)});b[l].l=+new Date;
+            e=o.createElement(i);r=o.getElementsByTagName(i)[0];
+            e.src='//www.google-analytics.com/analytics.js';
+            r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
+            ga('create','UA-XXXXX-X');ga('send','pageview');
```
Comment (aburgel) [inline index.html]: we don't have a good analytics account.

File: index.html [hunk 2]
```diff
@@ -1,53 +1,101 @@
...
+          <ul>
+            <li><a href="https://groups.google.com/group/django-non-relational" target="blank">Mailing list</a></li>
+            <li><a href="http://webchat.freenode.net/?channels=django-nonrel" target="blank">#django-nonrel</a> on Freenode</li>
+          </ul>
+        </li>
+      </ul>
     </div>
-  </div> <!-- /container -->
+    <footer>
+      <a href="http://www.djangoproject.com/"><img src="https://www.djangoproject.com/m/img/badges/djangoproject120x25.gif" border="0" alt="A Django project." title="A Django project." /></a>
```
Comment (aburgel) [inline index.html]: can we use this? we're not an official django project.

**Comment by aburgel** [comment]
looks good! but i'm not sure we should use the official django logo because we're not an official project.

also, where are we using FontAwesome?

**Comment by thegeorgeous** [comment]
will remove it then. Font Awesome is only used for the github icon in the
button. Should I remove that too?

**Comment by thegeorgeous** [comment]
@aburgel, have made the necessary changes. should I rebase them into one commit?

**Pull request closed without merging**